### PR TITLE
Port fix for #3075 to Release/1_0

### PR DESCRIPTION
**Pull request by int19h**

### CODE REVIEW

File: src/Package/Test/DataInspect/ViewersTest.cs [hunk 1]
```diff
@@ -18,19 +18,23 @@
...
+        private const REvaluationResultProperties AllFields = unchecked((REvaluationResultProperties)~0);
+
+        private readonly TestMethodFixture _testMethod;
         private readonly IRSessionProvider _sessionProvider;
         private readonly IObjectDetailsViewerAggregator _aggregator;
         private readonly IRInteractiveWorkflow _workflow;
 
-        public ViewersTest() {
+        public ViewersTest(TestMethodFixture testMethod) {
+            _testMethod = testMethod;
```
Comment (AlexanderSher) [inline src/Package/Test/DataInspect/ViewersTest.cs]: Do you need `TestMethodFixture` here?

Comment (int19h) [inline src/Package/Test/DataInspect/ViewersTest.cs]: I think it's not actually needed, this was from an earlier iteration when I was trying to start the session differently. But it's a port of the other commit that just went in, so I'd rather not touch anything here unless it's specifically required to merge into this branch.

**Comment by AlexanderSher** [comment]
👍 
Do we need a second reviewer?

**Comment by int19h** [comment]
@MikhailArkhipov ?

**Comment by MikhailArkhipov** [comment]
👍

**Pull request merged**